src/api/component.js

```diff
@@ -189,6 +191,7 @@ Component.prototype = Object.create(HTMLElement.prototype, {
   createdCallback: {
     configurable: true,
     value() {
+      setElementAsDefined(this);
```

Comment:
If it works in Chrome it should be fine. We'll do another breaking change soon to remove v0 support, so as long as the tests pass for it I'm okay with whatever works for now as long as we can get v1 fixed up.